lcd.lua

```diff
@@ -129,15 +129,29 @@ local clearscreen = function(pos)
 end
 
 local prepare_writing = function(pos)
-	local lcd_info = lcds[minetest.get_node(pos).param2]
-	if lcd_info == nil then return end
-	local text = minetest.add_entity(
-		{x = pos.x + lcd_info.delta.x,
-		 y = pos.y + lcd_info.delta.y,
-		 z = pos.z + lcd_info.delta.z}, "digilines_lcd:text")
-	text:setyaw(lcd_info.yaw or 0)
-	--* text:setpitch(lcd_info.yaw or 0)
-	return text
+	local existing
+	local objects = minetest.get_objects_inside_radius(pos, 0.5)
+	for _, o in ipairs(objects) do
+		local o_entity = o:get_luaentity()
+		if o_entity and o_entity.name == "digilines_lcd:text" then
+			existing = o_entity
+			break
+		end
+	end
+	if not existing or existing == nil then
```

Comment:
`not nil` is `true`. You don't need the extra `existing == nil`.
```suggestion
	if not existing then
```